salt/states/selinux.py

```diff
@@ -144,3 +161,59 @@ def boolean(name, value, persist=False):
         return ret
     ret['comment'] = 'Failed to set the boolean {0} to {1}'.format(name, rvalue)
     return ret
+
+
+def module(name, module_state='Enabled', version='any'):
+    '''
+    Enable/Disable and optionally force a specific version for an SELinux module
+
+    name
+        The name of the module to control
+
+    module_state
```

Comment:
Convention appears to be that all module names are lower case, but as best as I can tell this is not a requirement, nor enforced anywhere within selinux itself.  I recompiled and loaded one of my modules and switched the case around and all appears to be functional.

Rather than enforcing a convention that's not a technical requirement, I think documentation is the right approach.  Following what appears to be the convention elsewhere in salt, would tagging a "    # case sensitive" opposite name be appropriate (and sufficient?) here?